tests/test_state/test_matcha_state.py

```diff
@@ -82,6 +83,37 @@ def state_file_as_object() -> MatchaState:
     )
 
 
+@pytest.fixture
+def simplified_state_file_as_object(uuid_for_testing: uuid.UUID) -> MatchaState:
```

Comment:
This already exists in the fixture above, can't we re-use that with the addition of the `UUID`?